CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.0-beta.14 (2017-03-22)
+
```

Comment:
You should also mention the fix for loading SKY UX styles:

"SKY UX styles are now when running unit tests so CSS rules defined in SKY UX take effect during tests. This allows you to check the the expected computed style of an element when using things like the HTML `hidden` property that only take effect when SKY UX styles are loaded."